timers.js

```diff
@@ -1,4 +1,29 @@
+  var resetButton = document.querySelector("#reset");
+  var startButton = document.querySelector("#start");
+  var pauseButton = document.querySelector("#pause");
+  var stopWatch = document.querySelector("#timer");
+  var clicks = 0;
+  var seconds = 0;
 
+  function updateTime() {
+    stopWatch.innerHTML = ("Time Elapsed: " + seconds++);
+  };
 
 
+  startButton.addEventListener("click", function(){
+    clicks++;
+    if (clicks < 2) {
+      timerId = setInterval(updateTime, 1000);
+    }
+  });
 
+  pauseButton.addEventListener("click", function (){
+    clearInterval(timerId);
```

Comment:
For my previous comment to work, you'd need to set `timerId = false` after this line.